### DISCUSSION TITLE
Deploying images

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#695] [General]
I have a dev environment (my PC) and a test environment (azure). Both environments are point to the same database.  When I'm in Dev, I upload images and build some pages.  the images get uploaded to content/tenant/site/images.  if I publish dev to test, the images are broken.   I'm sure there is a relationship between the DB and the file uploaded.  I see the image names are routes, like http://localhost:44357/1/api/file/download/21 and somehow that finds my file.  So I trid setting the build-action to content for the files in my solution, but no dice.  I also don't seem to be able to re-upload an existing file, so I'm not sure what to try next. 

![image](https://user-images.githubusercontent.com/311063/90855678-0bdc9a00-e335-11ea-9fda-2c53633a89b3.png)
![image](https://user-images.githubusercontent.com/311063/90855705-17c85c00-e335-11ea-9535-239dde9fe7ac.png)

**Comment by sbwalker** [comment]
When you publish dev to test are you ensuring that the /content folder structure is being published? Have the verified the files exist in the test environment after deployment?
